graphics/src/widget/canvas/frame.rs

```diff
@@ -127,12 +130,17 @@ impl Frame {
                 lyon::math::Vector::new(size.width, size.height),
             );
 
-        let _ = lyon::tessellation::basic_shapes::fill_rectangle(
+        let mut tessellator = FillTessellator::new();
+        let options = FillOptions::default().with_fill_rule(rule.into());
+
+        let mut builder = tessellator.builder(&options, &mut buffers);
+
+        builder.add_rectangle(
             &lyon::math::Rect::new(top_left, size.into()),
-            &FillOptions::default().with_fill_rule(rule.into()),
-            &mut buffers,
-        )
-        .expect("Fill rectangle");
+            lyon::path::Winding::Positive,
+        );
+
+        let _ = builder.build().expect("Fill rectangle");
```

Comment:
I'm noticing a performance regression of around an order of magnitude in the `game_of_life` example.

I imagine the new `Frame::fill_rectangle` implementation is the culprit. We should try to address this before merging.

Reply:
The author of `lyon` addressed the issue in https://github.com/nical/lyon/pull/696 🎉